### DISCUSSION TITLE
Welcome to Proxmox Helper Scripts Discussions!

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by tonyva6** [comment]
Hi, I am trying to run the Proxmox VE 7.2-3 helper scripts on a NUC but they will not run, no errors.
Am I missing something?....Thanks

**Reply by tteck** [reply]
Sounds like you're missing internet.

**Reply by tteck** [reply]
Try running this in the Proxmox Shell
```
if nc -zw1 8.8.8.8 443; then  echo -e "Internet Connected"; else  echo -e "Internet NOT Connected"; fi
```

**Reply by tonyva6** [reply]
Hi, reply is Internet Connected

**Reply by tteck** [reply]
What about this
```
nslookup "github.com" | awk -F':' '/^Address: / { matched = 1 } matched { print $2}' | xargs
```

**Comment by tonyva6** [comment]
After about 60 seconds it returns nothing in the Proxmox Shell but I do notice at the bottom of the screen in Tasks and Status  a continually rotating circle.

**Reply by tteck** [reply]
You have DNS issues

It should have resolved to a IP instantly 
```
root@proxmox:~# nslookup "github.com" | awk -F':' '/^Address: / { matched = 1 } matched { print $2}' | xargs
140.82.113.4
```

**Reply by tonyva6** [reply]
I just tried to ping google.com but comes back with temporary failure in name resolution

**Reply by tteck** [reply]
Fix your DNS

**Reply by tonyva6** [reply]
I have fixed the DNS issue, thanks for your help.

**Comment by johndoe0815** [comment]
@tteck Do you have any backlog/plans that you can/want to share? There is another thread with over 200 posts with many ideas/requests for additional projects. But it is intransparent what you consider to provide, what you might be already working on, and what you will not do (too complicated or for whatever reason).
I like your offer very much and it makes proxmox usage really easy, without you I would not have set up so many containers that easy (nextcloudpi as one example, without your script a nightmare to get into proxmox). So do not take this question negative, i would just find it helpful to get more information on what to expect (so I can maybe wait for that) and what not (so I either need to use alternative tools or start trying to bring it into proxmox myself).
